### ISSUE TITLE
problems running example

### ISSUE THREAD

**Issue by ricoderks** [LocasaleLab/Automated-MFA-2023#2]
Hi, 

I just wanted to inform you that I had some problems running some of the code in the README file. 

I could not install all modules at first. I used python 3.8.20 via anaconda.

* The module `sklearn` doesn't exist anymore. Needed to install `scikit-learn`.
* Version 3.0 for `xlsxwriter` doesn't exist. I installed v3.0.1

After this the code:

`python main.py` 

ran.

The next code didn't run:

`python main.py computation experiments flux_analysis hct116_cultured_cell_line -t`

It asked for `openpyxl`. I installed it without supplying a version number. It installed version 3.1.5.

Code ran now.

Thanks for sharing your model / code.

Cheers, Rico

**Comment by liushiyu1994** [comment]
Thank you for your feedback. I’ve corrected the version labels, added the missing openpyxl dependency, and updated the README to emphasize the package needed when using Anaconda for installation. Your input is much appreciated!
